src/pages/participate/code.jsx

```diff
@@ -0,0 +1,352 @@
+import { Box, Typography, Link, List,ListItem, ListItemText, useTheme } from '@mui/material';
```

Comment:
You’re repeating a lot of similar List styles. You can extract that style into a constant or a styled component:

```tsx
const sharedListStyles = {
  fontFamily: '"Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif',
...
```

Then reuse it like:

```tsx
<List dense sx={sharedListStyles}>
```